### ISSUE TITLE
Debug (or explore alternatives to) Coveralls integration

### ISSUE THREAD

**Issue by jrgriffiniii** [samvera/hyrax#3811]
Coveralls seems to be in a position where it is no longer receiving code coverage analysis reports generated after the test suites are run on CircleCI: https://coveralls.io/github/samvera/hyrax.  https://github.com/samvera/hyrax/pull/3810 permits developers to generate the test coverage analysis locally (generating an HTML-formatted report), but this is far from ideal.

Thus far, I was not able to determine why this was failing (the configuration for the project on CircleCI seems to have the `COVERALLS_REPO_TOKEN` env. variable set properly).  One alternative to this might be to instead explore the usage of https://codeclimate.com/ for test coverage analysis.

**Comment by cjcolvar** [comment]
@jrgriffiniii FWIW Avalon uses code climate and we have some circleci config that deals with running tests in parallel which could probably be generalized and pulled into the orb.
https://github.com/avalonmediasystem/avalon/blob/master/.circleci/config.yml#L86-L128

**Comment by jrgriffiniii** [comment]
Thank you for this @cjcolvar, that will definitely be useful for testing.

**Comment by no-reply** [comment]
closing this as won't fix. i think we're doing our best with test coverage, and adding consistently.

if anyone is interested in working on the test suite in a directed way, a good first step would be for them to add the coverage tooling they need.
